CMakeLists.txt

```diff
@@ -35,18 +40,18 @@ endif()
 #===============================================================================
 if(dagmc)
   find_package(DAGMC REQUIRED)
-  link_directories(${DAGMC_LIBRARY_DIRS})
+  add_library(dagmc-imported INTERFACE IMPORTED)
+  target_link_directories(dagmc-imported INTERFACE ${DAGMC_LIBRARY_DIRS})
```

Comment:
This command is a little too shiny for our current required version of CMake (and the version on CI as well apparently). Could we maybe use `link_directories` here as we're only building one library? Another, perhaps more contained, option would be to set the `INTERFACE_LINK_DIRECTORIES` of the target directly.

Reply:
`link_directories` didn't quite work, but I figured out another way to do it (just add -L for each of the library dirs to `target_link_libraries`)